src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -103,7 +103,7 @@ public static IEnumerable<FrameworkInstallation> GetInstallations()
                 if (version != null && versionKey.GetInt("Install") == 1)
                 {
                     // 1.0 to 3.5
-                    Version.TryParse(version, out var parsed);
+                    _ = Version.TryParse(version, out var parsed);
```

Comment:
Is this some sort of analyzer we have that warns about this? I don't see it on my end and I'd say adding those `_` makes things more noisy.

Reply:
I get the warnings too. Visual Studio

Reply:
![image](https://user-images.githubusercontent.com/8229322/98404397-40fdac00-2049-11eb-897c-895f753d27c2.png)
The Sentry project is a bit noisy about unused return parameters, it compiles, but leaves a lot of error messages here. (I am running Visual Studio Community without any addon)